scripts/pre_commit_linter.py

```diff
@@ -140,7 +146,13 @@ def _pre_commit_linter():
     and pass JSCS binary path
     """
     parsed_args = _PARSER.parse_args()
-    autofix = bool(parsed_args.autofix)
+    input_dir = parsed_args.directory
+    if input_dir:
+        input_dir = os.path.join(os.getcwd(), parsed_args.directory[0])
+        if not os.path.isdir(input_dir):
+            print 'Please correct the path to the directory to be linted.Exiting.'
```

Comment:
'Could not locate directory %s. Exiting.'

(Having said that, it doesn't seem unreasonable for devs to pass in a single file as well. Maybe better to allow a filepath? If the filepath is a directory then this lints all files within this directory. If the filepath is a file then this lints just the file.)